service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsControllerAboutToSubmitTest.java

```diff
@@ -276,7 +312,7 @@ void shouldAdjournHearing(HearingReListOption adjournmentOption) {
         Element<HearingBooking> expectedAdjournedHearing = element(
             pastHearingToBeAdjourned.getId(),
             pastHearingToBeAdjourned.getValue().toBuilder()
-                .status(HearingStatus.ADJOURNED)
+                .status(adjournmentOption == RE_LIST_LATER ? ADJOURNED_TO_BE_RE_LISTED : ADJOURNED)
```

Comment:
I'd avoid to add if statements in tests. You can pass the expected value as a test parameter

Reply:
agree, did it this way as test is already parametrised and I dont really like @MethodSource to provide multiple params, happy to change